lib/variable/test/variable_test.cpp

```diff
@@ -122,6 +122,19 @@ TEST(Variable, is_readonly) {
   EXPECT_TRUE(Variable(const_var).is_readonly()); // propagated on copy
 }
 
+TEST(Variable, is_aligned_default) {
+  const auto var = makeVariable<double>(Values{1});
+  EXPECT_TRUE(var.is_aligned());
+}
+
+TEST(Variable, can_set_aligned_flag) {
+  auto var = makeVariable<double>(Values{1});
+  var.set_aligned(false);
+  EXPECT_FALSE(var.is_aligned());
+  var.set_aligned(true);
+  EXPECT_TRUE(var.is_aligned());
+}
+
```

Comment:
Missing tests for behavior under shallow and deep copy of variables, as well as assignment operators.

Reply:
We don't have those for readonly either

Reply:
This wasn't so much a reason for not doing it as for doing more and extending the other tests.